lib/luaxtable.c

```diff
@@ -43,37 +47,123 @@ typedef struct luaxtable_s {
 		struct xt_match match;
 		struct xt_target target;
 	};
+	int fallback;
 	luaxtable_type_t type;
 } luaxtable_t;
 
-static bool luaxtable_match(const struct sk_buff *skb, struct xt_action_param *par)
-{
-	return true;
-}
+static lunatik_object_t *xtable_targethooks = NULL;
+static lunatik_object_t *xtable_matchhooks = NULL;
```

Comment:
please, prefix your code with `luaxtable`, not `xtable`.. it makes it harder to track what is yours and what's original kernel.. btw, perhaps it's a good idea to structure it a bit.. but not sure.. I'll leave to you to think a bit more on this =)..

```
static struct {
	lunatik_object_t *target;
	lunatik_object_t *match;
} luaxtable_hooks = {NULL, NULL};
```